nautobot_floor_plan/choices.py

```diff
@@ -29,3 +29,15 @@ class AxisLabelsChoices(ChoiceSet):
         (NUMBERS, "Numbers"),
         (LETTERS, "Letters"),
     )
+
+
+class AllocationTypeChoices(ChoiceSet):
+    """Choices for tile allocation type."""
+
+    RACK = "RACK"
+    RACKGROUP = "RACKGROUP"
+
+    CHOICES = (
+        (RACK, "RACK"),
+        (RACKGROUP, "RACKGROUP"),
```

Comment:
Not critical, but by convention this would normally be more like:

```suggestion
    RACK = "rack"
    RACKGROUP = "rackgroup"

    CHOICES = (
        (RACK, "Rack"),
        (RACKGROUP, "Rack Group"),
```